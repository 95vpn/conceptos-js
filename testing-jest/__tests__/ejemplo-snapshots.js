const cliente = {
    nombre: "Johnny",
    balance: 500,
    tipo: 'premium'
};

describe('testing al cliente', () => {
    // test('Es Johnny', () => {
    //     expect(cliente.nombre).toBe('Johnny')
    // });
    test('Es Johnny', () => {
        expect(cliente).toMatchSnapshot()
    });
})