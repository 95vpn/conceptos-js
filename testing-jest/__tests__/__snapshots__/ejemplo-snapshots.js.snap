// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testing al cliente Es Johnny 1`] = `
{
  "balance": 500,
  "nombre": "Johnny",
  "tipo": "premium",
}
`;
